feature-group-definitions/web-animations.yml

```
@@ -0,0 +1,57 @@
+spec: https://drafts.csswg.org/web-animations/
```

Tests are failing because this URL isn't in browser-specs. This will fix it:

```suggestion
spec: https://drafts.csswg.org/web-animations-1/
```